docs/_posts/2016-09-05-pouchdb-6.0.0.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+
+title: PouchDB 6.0.0: Labour Day
+author: Dale Harvey
+
+---
+
+To celebrate the USA and Canada's Labo(u)r Day we have decided to work extra hard to bring
+you PouchDB 6.0.0 which brings early retirement to those features that have done great
```

Comment:
It would be good to have a comma before "which", or alternatively split it up into two sentences, e.g.
```
"[…] bring you PouchDB 6.0.0. It brings early retirement […]"
```